utils/src/shutdown.rs

```diff
@@ -71,6 +73,13 @@ impl Trigger {
             .store(true, Ordering::Relaxed);
     }
 
+    /// Mark leader notified
+    #[inline]
+    pub fn mark_leader_notified(&self) {
+        debug!("mark leader notified");
```

Comment:
It seems we should use `info!` rather than `debug!` here since the shutdown message is important.

Reply:
Updated